### PR TITLE
Add Cronicle

### PR DESCRIPTION
**Pull request by Ismaaa**
Added [Cronicle](https://github.com/jhuckaby/Cronicle) 😄 

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

**Comment by githubsaturn** [comment]
Where does it store the data? Restart you instance and all your data is gone.

**Comment by Ismaaa** [comment]
Thanks @githubsaturn !
Just updated the file with the missing volumes

File: public/v4/apps/cronicle.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: soulteary/cronicle:$$cap_version
+        restart: always
+        volumes:
+            - /etc/localtime:/etc/localtime:ro
+            - /etc/timezone:/etc/timezone:ro
+            - $$cap_appname:/opt/cronicle/data:rw
+            - $$cap_appname-plugins:/opt/cronicle/plugins:rw
```
Comment (githubsaturn) [inline public/v4/apps/cronicle.yml:10]: Thanks!

CapRover doesn't read the permission modes `:ro` etc... I'll update after merge.

Comment (Ismaaa) [reply public/v4/apps/cronicle.yml:10]: Oh, thanks! Didn't know that! 😃

**Comment by uhlhosting** [comment]
@Ismaaa can you tell me how you add ENV variables please? Also existing TZ variable is not working.

**Comment by coffseducation** [comment]
I cannot get this app to work on Ubuntu 22.04

   Error: Script failed: No such file or directory (spawn /tmp/cronicle-script-temp-jlreez7cc01.sh ENOENT

**Comment by Ismaaa** [comment]
> @Ismaaa can you tell me how you add ENV variables please? Also existing TZ variable is not working.

You'll have to modify the file, customize it and then deploy it as a custom template.
You can add as many variables as you need here:

```yml
        environment:
            - TZ=$$cap_timezone
            - MY_VAR=hello
```

**Comment by Ismaaa** [comment]
> I cannot get this app to work on Ubuntu 22.04
> 
> Error: Script failed: No such file or directory (spawn /tmp/cronicle-script-temp-jlreez7cc01.sh ENOENT

Just tested it in Ubuntu 22.04 and it works perfectly for me.
Can you please give me more info? Are you running it in x86 or ARM?

**Comment by uhlhosting** [comment]
In the end i managed to have it working.

**Comment by Ismaaa** [comment]
Glad that you fixed it.
What was the issue and how did you solve it?

**Pull request merged**